src/client/pmix_client_fence.c

```diff
@@ -200,7 +201,12 @@ static pmix_status_t unpack_return(pmix_buffer_t *data)
     
     /* provide an opportunity to store any data (or at least how to access
      * any data) that was included in the fence */
-    PMIX_GDS_RECV_MODEX_COMPLETE(data);
+    // TODO(skg) Is this peer correct? It seems to work.
```

Comment:
yes, that is the correct peer - sorry for the error!

Reply:
Thank you. Just making sure. I removed the comment.